From 7154d371c83df8fa6b9ffe4756b08361c7ff3227 Mon Sep 17 00:00:00 2001
From: Patrick Griffis <pgriffis@igalia.com>
Date: Thu, 10 Apr 2025 09:12:28 -0700
Subject: [PATCH] [WPE] Remove unnecessary introspection annotations
 https://bugs.webkit.org/show_bug.cgi?id=291354

Reviewed by Michael Catanzaro.

This used the copy-func and free-func annotations which was
added in gobject-introspection 1.74 (1 year newer than our
glib dep 2.70).

However this struct is a GBoxed type so introspection already
knows this information and it can be removed.

* Source/WebKit/WPEPlatform/wpe/WPEBufferDMABufFormats.cpp:

Canonical link: https://commits.webkit.org/293523@main
---
 Source/WebKit/WPEPlatform/wpe/WPEBufferDMABufFormats.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebKit/WPEPlatform/wpe/WPEBufferDMABufFormats.cpp b/Source/WebKit/WPEPlatform/wpe/WPEBufferDMABufFormats.cpp
index 9a2a8cdf7a2e..30dc108f17bf 100644
--- a/Source/WebKit/WPEPlatform/wpe/WPEBufferDMABufFormats.cpp
+++ b/Source/WebKit/WPEPlatform/wpe/WPEBufferDMABufFormats.cpp
@@ -225,7 +225,7 @@ GArray* wpe_buffer_dma_buf_formats_get_format_modifiers(WPEBufferDMABufFormats*
 }
 
 /**
- * WPEBufferDMABufFormatsBuilder: (copy-func wpe_buffer_dma_buf_formats_builder_ref) (free-func wpe_buffer_dma_buf_formats_builder_unref)
+ * WPEBufferDMABufFormatsBuilder:
  *
  * Helper type to build a #WPEBufferDMABufFormats
  */
-- 
2.43.0

